src/main/java/org/owasp/esapi/configuration/EsapiPropertyLoaderFactory.java

```diff
@@ -51,8 +51,8 @@ public static AbstractPrioritizedPropertyLoader createPropertyLoader(EsapiConfig
         if (PROPERTIES.getTypeName().equalsIgnoreCase(fileExtension)) {
             return new StandardEsapiPropertyLoader(cfgPath, cfg.getPriority());
         } else {
-            throw new ConfigurationException("Configuration storage type [" + fileExtension + "] is not " +
-                    "supported");
+            throw new ConfigurationException("The extension of given configuration path [ " + cfgPath + " ] is not supported." +
+                    "\nOnly .xml or .properties file extensions are supported.");
```

Comment:
Minor change request. Either remove the '\n' here entirely or replace it with the value of
```java
System.getProperty("line.separator")
```
so that it works as intended one *nix and Windows. Personally, I'd just remove the newline and replace it with a space as that make it easier to grep logged exceptions.

Reply:
Sure thing! Should be quick.

Reply:
I'm sorry for such a delay. I'm still new to git and github and this is my first pull request ever. I just finished my first year of college and all projects I have done were in a single branch and weren't forked, however I have already changed what you told me to change.

Reply:
@HenriquePinto333 - No worries. That's how we all learn. Took me way too long to get used to git after using at least a half-dozen other version control systems.